client/src/components/list/_UserList.tsx

```diff
@@ -80,9 +86,12 @@ const UserList: React.FC<UserListProps> = ({ userlist, isOwn }) => {
         coverImage: anilistMedia.coverImage.medium,
         bannerImage: anilistMedia.bannerImage
       }));
-
+      
       setListItems(
-        prev => prev.concat(listItems)
+        prev => {
+          const ids = new Set<number>(prev.map(item => item.id));
+          return page === 1 ? listItems : prev.concat(listItems.filter(item => !ids.has(item.id)));
```

Comment:
could you explain why we check if the page is 1 here?

Reply:
Oops I think it's leftover from previous nonworking implementation and it's not really needed